sharing_portal/views.py

```diff
@@ -656,10 +687,18 @@ def list_daypass_requests(request, **kwargs):
         for project in keycloak_client.get_user_roles(request.user.username)
         if manage_membership_in_scope(project["scopes"])
     ]
+    trovi_artifacts = trovi.list_artifacts(request.session.get("trovi_token"))
```

Comment:
It would be nice to one day be able to pass more filters to the ListArtifacts endpoint 😔